protocol/src/main/java/org/fidoalliance/fdo/protocol/KeyResolver.java

```diff
@@ -93,38 +98,74 @@ private void generateKey(PublicKeyType keyType,
     }
   }
 
+  /**
+   * Gets the underlying keystore.
+   * @return The underlying keystore.
+   */
+  public KeyStore getKeyStore() {
+    return this.keyStore;
+  }
+
   /**
    * Loads the keys to be resolved.
+   *
    * @param config The Keystore description.
    * @throws IOException An error occured.
    */
   public void load(KeyStoreConfig config) throws IOException {
-    try {
-      this.config = config;
-      this.keyStore = KeyStore.getInstance(config.getStoreType());
+    this.config = config;
+    if (config.getStoreType().equals("PEM")) {
 
-      String path = config.getPath();
-      if (path != null) { // we have a stream path to load from
-        try (InputStream input =
-            Config.getWorker(KeyStoreInputStreamFunction.class).apply(path)) {
-          keyStore.load(input, getPasswordArray());
+      //load from pem
+
+      try {
+        String pemString = Files.readString(Path.of(config.getPath()));
+        List<Certificate> certs = PemLoader.loadCerts(pemString);
+        String keyPass = config.getPassword();
+        PrivateKey privateKey = PemLoader.loadPrivateKey(pemString,keyPass);
+
+        this.keyStore = KeyStore.getInstance("JKS");
```

Comment:
Are we loading the Keystore from a `.jks` format file or .pem readable file? Instead of keeping key pair in a .pem file; can we load it as container secret ?